docs/flow360doc/component/flow360_params.rst

```diff
@@ -0,0 +1,38 @@
+flow360 solvers parameters
```

Comment:
how are these files generated? Automatically or manually?

Reply:
The rst files are written manually, but the documentation are generated automatically.